test/stdlib/PrintFloat.swift.gyb

```diff
@@ -592,7 +592,9 @@ PrintTests.test("Printable_Float") {
   expectDescription("-inf", -Float.infinity)
   expectDescription("3.1415925", Float.pi)
   expectDescription("3.4028235e+38", Float.greatestFiniteMagnitude)
+#if !arch(arm)
   expectDescription("1e-45", Float.leastNonzeroMagnitude)
+#endif
   expectDescription("1.1754944e-38", Float.leastNormalMagnitude)
```

Comment:
I might instead test `leastNonzeroMagnitude` only if it is in fact a different value:
```
  if Float.leastNonzeroMagnitude != Float.leastNormalMagnitude {
     expectDescription("1e-45", Float.leastNonzeroMagnitude)
  }
```

Reply:
(Also, I don't *think* that we have any other platforms where subnormals are disabled, and there's something to be said for having test failures if someone tries to bring up another platform where they are, though I would prefer that the tests be more explicitly about that.)